client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -181,28 +175,43 @@ h2.action-panel__title {
 	display: none;
 }
 
+.email-providers-stacked-comparison__main.main .card.action-panel.promo-card.email-providers-stacked-comparison__provider-card .action-panel__figure {
+	margin-right: unset;
+	@include break-mobile {
+		margin-right: 24px;
+	}
+}
+
+.email-provider-stacked-card__provider-expand-cta {
+	width: 110px;
+}
+
 .is-expanded {
 	.email-provider-stacked-card__provider-expand-cta {
 		display: none;
 	}
 
 	.email-provider-stacked-card__provider-form-and-right-panel {
 		display: flex;
-		flex-direction: column-reverse;
+		flex-direction: column;
 		justify-content: start;
 		margin-top: 1em;
 		line-height: 1.5;
 		padding-right: 20px;
 
-		@include breakpoint-deprecated( '>1040px' ) {
+		@include break-xlarge {
 			flex-direction: row;
 		}
 	}
 
 	.email-provider-stacked-card__provider-form {
 		margin-bottom: 20px;
 		margin-right: 20px;
-		width: $width-left-panel-card;
+		width: 100%;
+
+		@include break-xlarge {
+			width: $width-left-panel-card;
+		}
 
 		@include breakpoint-deprecated( '>1040px' ) {
```

Comment:
Should this breakpoint also be updated to match the others we're adding in this PR?